Backend/src/routes/api.js

```diff
@@ -16,12 +17,13 @@ ROUTER.get("/api", (req, res) => {
 });
 
 //----------------------------------------------------- TCP ----------------------------------------------------------
-const CAR_PORT = 4003; // Port for TCP connection
-const CAR_SERVER = "localhost"; // TCP server's IP address (Replace with pi's IP address to connect to pi)
+const CAR_PORT = IMPORTED_CONSTANTS.CAR_PORT; // Port for TCP connection
+const CAR_SERVER = IMPORTED_CONSTANTS.CAR_SERVER; // TCP server's IP address (Replace with pi's IP address to connect to pi)
```

Comment:
Good time to fix the bad naming convention I had previously used.
```suggestion
const CAR_PORT = CONSTANTS.CAR_PORT; // Port for TCP connection
const CAR_ADDRESS = CONSTANTS.TEST_ADDRESS; // TCP server's IP address (Replace with PI_ADDRESS to connect to pi)
```